fury/colormap.py

```diff
@@ -591,3 +591,15 @@ def _generate_next_color():
         return [c for i, c in zip(range(nb_colors), _generate_next_color())]
 
     return _generate_next_color()
+
+
+def hex_to_rgb(color):
```

Comment:
Can you add the documentation of the function. See [this function as example](https://github.com/fury-gl/fury/blob/master/fury/colormap.py#L208)

Reply:
Done.